src/k8s/pkg/k8sd/setup/kube_apiserver_test.go

```diff
@@ -245,4 +245,30 @@ func TestKubeAPIServer(t *testing.T) {
 		g.Expect(err).To(HaveOccurred())
 		g.Expect(err).To(MatchError(ContainSubstring("unsupported datastore")))
 	})
+
+	t.Run("IPv6", func(t *testing.T) {
+		g := NewWithT(t)
+
+		// Create a mock snap
+		s := mustSetupSnapAndDirectories(t, setKubeletMock)
+		s.Mock.Hostname = "dev"
+
+		g.Expect(setup.KubeAPIServer(s, net.ParseIP("2001:db8::"), "fd98::/108", "https://auth-webhook.url", false, types.Datastore{Type: utils.Pointer("k8s-dqlite")}, "Node,RBAC", nil)).To(Succeed())
```

Comment:
Just a random question. Are these IPs supposed mean anything?

Reply:
`2001:db8::` is the IP used in [docs](https://docs.oracle.com/cd/E19253-01/816-4554/ipv6-overview-170/index.html#:~:text=The%20prefix%202001%3Adb8%3A%3A%2F,has%20the%20following%20subnet%20prefix.&text=The%20subnet%20prefix%20always%20contains%2064%20bits.)
`fd98::/108` is the CIDR we use in our examples/integration tests.